pkg/application/application.go

```diff
@@ -41,97 +32,37 @@ func GetDefaultAppName(existingApps []preference.ApplicationInfo) (string, error
 
 	// If there's no prefix in config file or it is equal to $DIR, use safe default which is the name of current directory
 	if cfg.OdoSettings.NamePrefix == nil || *cfg.OdoSettings.NamePrefix == "" {
-		prefix, err := component.GetComponentDir("", occlient.NONE)
+		prefix, err := component.GetComponentDir("", util.NONE)
 		if err != nil {
 			return "", errors.Wrap(err, "unable to generate random app name")
 		}
-		appName, err = util.GetRandomName(prefix, appPrefixMaxLen, existingAppNames, appNameMaxRetries)
+		appName, err = util.GetRandomName(prefix, appPrefixMaxLen, []string{}, appNameMaxRetries)
 		if err != nil {
 			return "", errors.Wrap(err, "unable to generate random app name")
 		}
 	} else {
-		appName, err = util.GetRandomName(*cfg.OdoSettings.NamePrefix, appPrefixMaxLen, existingAppNames, appNameMaxRetries)
+		appName, err = util.GetRandomName(*cfg.OdoSettings.NamePrefix, appPrefixMaxLen, []string{}, appNameMaxRetries)
 	}
 	if err != nil {
 		return "", errors.Wrap(err, "unable to generate random app name")
 	}
 	return util.GetDNS1123Name(appName), nil
 }
 
-// Create a new application
-func Create(client *occlient.Client, appName string) error {
-
-	exists, _ := Exists(client, appName)
-
-	if exists {
-		return fmt.Errorf("unable to create new application, %s application already exists", appName)
-	}
-
-	cfg, err := preference.New()
-	if err != nil {
-		return errors.Wrap(err, "unable to create new application")
-	}
-
-	err = cfg.AddApplication(appName, client.Namespace)
-	if err != nil {
-		return errors.Wrap(err, "unable to create new application")
-	}
-	return nil
-}
-
-// List all applications in current project
```

Comment:
The comment of the function is removed too

Reply:
Ack.. Fixed it now